backend/src/data/migrations/20220906162845_add_transactions_table.ts

```diff
@@ -0,0 +1,59 @@
+import { Knex } from 'knex';
+
+enum TableName {
+  TRANSACTIONS = 'transactions',
+  USERS = 'users',
+}
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  SENDER_ID = 'sender_id',
+  RECEIVER_ID = 'receiver_id',
+  AMOUNT = 'amount',
+  STATUS = 'status',
+}
+
+enum TransactionStatus {
+  PENDING = 'pending',
+  HOLD = 'hold',
```

Comment:
what are the differences?

Reply:
Pending is the status when our transaction is sent. Hold is the status which is set when users funds are hold, waiting for being payed to mentor

Reply:
Pending - is the status, which transaction gets when it starts its transfering, I would say like that